src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs

```diff
@@ -49,7 +49,8 @@ public class HostJsonFileConfigurationProvider : ConfigurationProvider
             private static readonly string[] WellKnownHostJsonProperties = new[]
             {
                 "version", "functionTimeout", "functions", "http", "watchDirectories", "queues", "serviceBus",
-                "eventHub", "singleton", "logging", "aggregator", "healthMonitor", "extensionBundle", "managedDependencies"
+                "eventHub", "singleton", "logging", "aggregator", "healthMonitor", "extensionBundle", "managedDependencies",
```

Comment:
The pattern we use now for logging config options is the IOptionsFormatter mechanism, e.g. https://github.com/Azure/azure-webjobs-sdk/blob/6a6b0d0c3c03282f4be5752fb26f0aef1e154b9c/src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Config/QueuesOptions.cs#L17. I think this WellKnownHostJsonProperties thing is obsolete and we should probably remove it. @brettsam ?

Reply:
Yes, this really doesn't apply anymore. It's now up to the Options themselves to tell us what is relevant for logging. This is so we know the "truth" about what options are being used (and allows each extension to tell us what to log in their own code without having to map something here). 

I think this whole thing should go away...